src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -216,28 +216,40 @@ internal bool ShouldRunNullableWalker
             {
                 if (!_lazyShouldRunNullableWalker.HasValue())
                 {
+                    _lazyShouldRunNullableWalker = calculateResult();
+                }
+                return _lazyShouldRunNullableWalker.Value();
+
+                ThreeState calculateResult()
+                {
+                    var feature = Feature("nullableAnalysis");
```

Comment:
>nullableAnalysis [](start = 43, length = 16)

It is confusing to have this new "nullable-analysis" switch (used for all calls to `NullableWalker`) in addition to the existing "run-nullable-analysis" switch (used for calls to `NullableWalker` from `SemanticModel` only).

I'll re-purpose "run-nullable-analysis" to cover all calls. And I'll change the expected values from `{ true, false }` to `{ always, never }` to clarify the use:

```
run-nullable-analysis         // equivalent to no switch
run-nullable-analysis=always  // analyze all methods
run-nullable-analysis=never   // skip analysis of methods
run-nullable-analysis=<other> // silently ignored; equivalent to no switch
```